prowler/providers/gcp/gcp_provider.py

```diff
@@ -351,6 +368,24 @@ def setup_session(
                         file=__file__, original_exception=error
                     )
 
+            if service_account_key:
```

Comment:
`if` or `elif`?

Reply:
The elif is not needed, If it matches any statement the return will be triggered.